lib/config.js

```diff
@@ -460,6 +464,13 @@ Config.prototype.update = function (options) {
     this.channel = ''
   }
 
+  if (this.targetOS === 'android') {
+    this.androidChannel = this.channel
+    if (this.androidChannel === '') {
+      this.androidChannel = 'stable'
```

Comment:
Wasn't sure this was acceptable, but it's needed to satisfy the assertion here: https://cs.chromium.org/chromium/src/chrome/android/channel.gni?type=cs&g=0&l=11.